libbeat/outputs/mode/lb/async_worker.go

```diff
@@ -92,7 +92,7 @@ func (w *asyncWorker) connect() bool {
 			return false
 		}
 
-		debugf("connect failed with: %v", err)
+		logp.Info("Connect failed with: %v", err)
```

Comment:
this will print the info message after waiting for Sleep on backoff handler. Alternatively:

```
for {
    err := w.client.Connect(w.ctx.timeout)
    if err == nil {
        w.backoff.Reset()
        return false
    }
    
    logp.Info("Connect failed with: %v", err)

    stopped := w.backoff.Wait()
    if stopped {
        return true
    }
}
```